File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// VogelwarteSchweiz SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class VogelwarteSchweizMakeContext
{
    public static function call(array $ctxmap, ?VogelwarteSchweizContext $basectx): VogelwarteSchweizContext
    {
        return new VogelwarteSchweizContext($ctxmap, $basectx);
    }
}

File: .sdk/tm/rb/utility/make_context.rb
# VogelwarteSchweiz SDK utility: make_context
require_relative '../core/context'
module VogelwarteSchweizUtilities
  MakeContext = ->(ctxmap, basectx) {
    VogelwarteSchweizContext.new(ctxmap, basectx)
  }
end
